crates/experimentation_client/src/types.rs

```diff
@@ -29,10 +30,16 @@ pub(crate) enum VariantType {
 #[derive(Deserialize, Serialize, Clone, Debug)]
 pub struct Variant {
```

Comment:
DISCUSSION Point: This type also needs to be unified, just like `Config` type was unified in this PR

Reply:
Raise in github discussions.